tests/test_apple.py

```diff
@@ -84,7 +84,7 @@ def testPickCenters(self):
         apple_picker = Apple()
```

Comment:
I looked at the code and seems like unless we pass in `output_dir` to this constructor, the .jpg will not actually be created at all. So I'm wondering how you were able to test this out `test_apple` locally

Reply:
OOPS. I run the apple.py command for my case which passes the output folder. I will fix this.

Reply:
Added the `tests` folder as the output folder in the commit d07f804.

Reply:
So let's pass it to the constructor then instead of assigning the attribute.

Reply:
@junchaoxia .. this is a common paradigm when testing things .. if you want to create a temporary folder to test things out, create it using one of the `tempfile` module functions, use it (in this case pass it in to the `Apple` constructor), and then clean it up. You don't know what `Apple` is going to do - run it the first time - it works. Run it again, it may behave differently because it doesn't want to overwrite your existing file. Also, you're giving it your `DATA_DIR` to write in, which you want to preserve from any changes anyway.

Reply:
Thanks. Changed to generate the temporary folder for output the test image in new commit  7216539.